src/app/Pages/DetailsPage/index.jsx

```diff
@@ -104,9 +108,12 @@ export default class DetailsPage extends Component {
         <div className="lbh-container row details">
           <div className="details__left-column">
             <PersonalDetails customer={this.state.customer} id={customerId} />
-            {hasFeatureFlag('things-to-note') && (
-              <ThingsToNote customerId={customerId} />
-            )}
+            {isMemberOfGroups([
```

Comment:
Had been wondering whether it would be easier for reuse to implement this in a more react way by making this a component, e.g.

```
<ShowForGroups groups={['HOUSING_NEEDS_SV', 'HOUSING_NEEDS']}>
     <ThingsToNote customerId={customerId} />}
</ShowForGroups>
```